### PR TITLE
feat: adding active and hover states to search input

### PR DESCRIPTION
**Pull request by Kira-Pilot**
Couple of improvements:
1. get rid of highlight when autosuggestions are on. 
Before:
Screen Shot 2022-06-29 at 5 11 22 PM

After:
Screen Shot 2022-06-29 at 4 56 33 PM

2. add hover and active states
Kapture 2022-06-29 at 17 05 42

### CODE REVIEW

File: site/src/components/SearchBarWithFilter/SearchBarWithFilter.tsx [hunk 1]
```diff
@@ -106,19 +106,15 @@ export const SearchBarWithFilter: React.FC<SearchBarWithFilterProps> = ({
         )}
 
         <form onSubmit={form.handleSubmit} className={styles.filterForm}>
-          <TextField
+          <OutlinedInput
```
Comment (Kira-Pilot) [inline site/src/components/SearchBarWithFilter/SearchBarWithFilter.tsx:109]: Changed this because TextField is a nightmare to style.

**Review by kylecarbs — APPROVED** [review]
This is pretty!

**Review by AbhineetJain — APPROVED** [review]
This is nice! Does `OutlinedInput` also have an error state? Maybe we can use a *reddish* border when the query is erroneous.

**Comment by Kira-Pilot** [comment]
> This is nice! Does `OutlinedInput` also have an error state? Maybe we can use a _reddish_ border when the query is erroneous.

Yes, it should! Working on that now after looking at the test failures.

**Comment by Kira-Pilot** [comment]
@AbhineetJain We are using the same outline colors! I think it's a standard for MUI inputs but idk the whole palette history, tbh:
Screen Shot 2022-06-30 at 11 05 27 AM
You're right about the error text - that is a bit darker - but I think that's okay. I actually tried to change it to match, but it showed up white? Idk what they're doing behind the scenes at MUI but considering we might move away from that library in the future, I'm not too worried about it.

File: site/src/components/SearchBarWithFilter/SearchBarWithFilter.tsx [hunk 2]
```diff
@@ -152,29 +149,39 @@ export const SearchBarWithFilter: React.FC<SearchBarWithFilterProps> = ({
   )
 }
 
-const useStyles = makeStyles((theme) => ({
+interface StyleProps {
+  error?: unknown
```
[the remark this answers is not in the document]
Comment (Kira-Pilot) [reply site/src/components/SearchBarWithFilter/SearchBarWithFilter.tsx:153]: Good catch! I'm putting up another PR right after this one with more style improvements and I promise I'll fix it there. 🤞

**Pull request merged**